introduction.md

```diff
@@ -1,2 +1,3 @@
 favorite Code school path : Git, so far ('cause it's the only one)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?